Fix optional performer and non-optional period

stevenvegt opened the pull request:
Fix #9

stevenvegt committed:
Fix optional performer and non-optinal period

woutslakhorst commented on pkg/consent-logic_test.go:
```diff
@@ -589,3 +589,43 @@ func Test_hashFHIRConsent(t *testing.T) {
+	}
+	consentLogic := &ConsentLogic{NutsRegistry: registryMock, NutsCrypto: cryptoMock}
+	event, err := consentLogic.buildConsentRequestConstructedEvent(createConsentRequest)
+	if err != nil {
```
It would be a good moment to introduce testify assertions

stevenvegt committed:
Introduce testify

Introduce testify/assert package.

Only used in the newly added tests. Later other test could be rewritten to use testify assertions.
